util/xcode/install_resources.cmake

```diff
@@ -0,0 +1,63 @@
+
+# This is only one working solution I found to be working for normal and Archive builds under Xcode 15.4
+# I expect higger sensitivity to Xcode version.
```

Comment:
This file is at high risk of being outdated:

1. Duplication. Most of the paths and file lists are already contained in `src/CMakeLists.txt`. They could either be imported, or this file merged into the main file.
2. None of commands in this file are tested on anything but Xcode. Out-of-sync situations are not found automatically.

Reply:
I do not know the solution how to merge this into `src/CMakeLists.txt` and keep it in a working state.
The problem there is that CMake has no idea about the "Archive" build type in Xcode and does not know where the app is being packaged. I don't find a working way, how to say to CMake to add Resources in a way, that they do not lose the directory structure. Everything that I found that should work, did not work.

It can be checked manually by the dir diff tools applied for Resources in the app Bundle from CMake and Xcode.

I try to add some CI checks for it. Hopefully, it will work.

Reply:
Now it is verified by CI run and directory diff.